CONTRIBUTING.md

```diff
@@ -104,9 +104,10 @@ your PR. People who review your PR are happy to know
 - Press `Create pull request`. Now you can navigate through your PR, which
 highlights the changes you made.
 
-Pull Requests DO NOT have to be large: it is much easier to review small
-targeted PRs than a huge chunk of code, so feel free to split your work
-in small pieces.
+Please DO NOT write large Pull Requests, as they are very difficult and
+time-consuming to review. As much as possible, split them into small
+targeted PRs. If you want to implement a feature, and are not too sure how
+to split it, feel free to ping other developers.
```

Comment:
I think that's great advise but too abstract for new contributors that have not developed in teams before. Let's also add an example.

```suggestion
targeted PRs.
For example, if find typos in the documentation open a pull request that only fixes typos.
If you want to fix a bug, make a small pull request that only fixes a bug.
If you want to implement a large feature, write helper functionality first, test it and submit those as a first pull request.
If you want to implement a feature and are not too sure how to split it, just open an issue about your plans and ping other WarpX developers on it to chime in.
```

Reply:
Sounds great, Thank you for this relevant multi-line suggestion :).